src/pages/HomePage/HomePage.tsx

```diff
@@ -45,7 +46,7 @@ const HomePage: React.FC = (): any => {
     // console.log(newValue.value);
   };
 
-  return ctx.user ? (
```

Comment:
@juroc95 We are temporarily disabling the "is user logged in?" check because it was misbehaving but we will put it back ASAP.

Reply:
If you have questions, tag Alana or Nathan because my understanding of this is very basic.

Reply:
Traveling between different pages has to be done through button pressing (ex. history.push("/app/something")). If the user tries to direct to a different page by entering the URL manually, it will trigger the user to log out.

Reply:
In that case, the auth check can be added back in once the home page is more integrated.